unit/Makefile

```diff
@@ -23,6 +23,8 @@ SRC += analyses/ai/ai.cpp \
        analyses/variable-sensitivity/last_written_location.cpp \
        big-int/big-int.cpp \
        compound_block_locations.cpp \
+       get_goto_model_from_c_test.cpp \
+       goto-instrument/cover_instrument.cpp \
```

Comment:
Noting this line is different from original commit

Reply:
:no_entry_sign: in fact - this shouldn't be here, the line is included two lines down

Reply:
Good spot, fixed